platform-tests/example-apps/healthcheck/db.go

```diff
@@ -35,7 +44,7 @@ func testDBConnection(ssl bool) error {
 		dbURL.RawQuery = dbURL.RawQuery + "&sslmode=disable"
 	}
 
-	db, err := sql.Open("postgres", dbURL.String())
+	db, err := sql.Open(service, dbURL.String())
```

Comment:
I'm not sure how this can work for mysql without the mysql driver being imported.